specifications/README.md

```diff
@@ -53,7 +53,7 @@ As [depicted](#The-Diem-network), there are 4 network protocol specifications:
 
 In addition, these specifications build on top of a common set of specifications:
 
-* **[Diem canonical serialization (LCS)](https://developers.diem.com/docs/rustdocs/diem_canonical_serialization/)**. This details how data is canonically serialized and deserialized. Hashing and signing of data structures are applied after LCS serialization.
+* **[Diem canonical serialization (BCS)](https://developers.diem.com/docs/rustdocs/diem_canonical_serialization/)**. This details how data is canonically serialized and deserialized. Hashing and signing of data structures are applied after BCS serialization.
```

Comment:
Should the doc file in the URL be named to .../rustdocs/binary_canonical_serialization?